code/modules/events/wizard/magicarp.dm

```diff
@@ -38,9 +38,7 @@
 	health = 50
 	gold_core_spawnable = NO_SPAWN
 	random_color = FALSE
-	food_type = list()
-	tame_chance = 0
-	bonus_tame_chance = 0
+
```

Comment:
```suggestion
```

Reply:
i need to override this differently but havnt yet which is why i havnt removed these.